src/vmm/src/logger/mod.rs

```diff
@@ -44,3 +44,11 @@ pub fn update_metric_with_elapsed_time(metric: &SharedStoreMetric, start_time_us
     metric.store(delta_us);
     delta_us
 }
+
+/// Helper function for updating the value of an inc metric with elapsed time since some time in a
+/// past.
+pub fn update_inc_metric_with_elapsed_time(metric: &SharedIncMetric, start_time_us: u64) -> u64 {
```

Comment:
If we don't want min/max for the metrics then I would prefer to modify [LatencyMetricsRecorder](https://github.com/sudanl0/firecracker/blob/be25214c564812babbc614ad2e4077afd0bba4a3/src/vmm/src/logger/metrics.rs#L695) to use generic data types so that it can work for LatencyAggregateMetrics and SharedIncMetrics.

Reply:
removed.